RELEASE_NOTES

```diff
@@ -1,6 +1,20 @@
 This file contains a description of the major changes to the EESSI
 build-and-deploy bot. For more detailed information, please see the git log.
 
+v0.1.1 (14 november 2023)
+--------------------------
+
+This is a minor release of the EESSI build-and-deploy bot.
```

Comment:
strictly speaking it's a patch release (minor would be v0.2.0, major would be v1.0.0), but I don't want to get too pedantic ;)